lib/vdsm/supervdsm_api/virt.py

```diff
@@ -102,16 +102,30 @@ def mdev_create(device, mdev_type, mdev_uuid=None):
     """
     path = os.path.join(
         '/sys/class/mdev_bus/{}/mdev_supported_types/{}/create'.format(
-            device, mdev_type
+            device, mdev_properties.device_type
         )
     )
-
     if mdev_uuid is None:
         mdev_uuid = str(uuid.uuid4())
-
     with open(path, 'w') as f:
         f.write(mdev_uuid)
 
+    driver_parameters = mdev_properties.driver_parameters
+    if driver_parameters is not None:
+        param_path = f'/sys/bus/mdev/devices/{mdev_uuid}/nvidia/vgpu_params'
+        try:
+            with open(param_path, 'w') as f:
+                f.write('%s\n' % (driver_parameters,))
```

Comment:
Maybe worth adding the comment that additional character is required.

Reply:
What exactly does "additional character" mean? Is it new line? Is it space? Is it arbitrary character (meaning last character is always lost)? ... The commit message is not making this clear either.